Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -504,17 +552,22 @@ def send_slack_request_sync(client: slack_sdk.WebClient, method: str, http_verb:
     return response  # type: ignore
 
 
-async def send_slack_request_async(client: AsyncWebClient, method: str, http_verb: str = 'POST', file_: str = '',
-                                   body: dict = None) -> SlackResponse:
+async def send_slack_request_async(
+    client: AsyncWebClient,
+    method: str = '',
+    http_verb: str = 'POST',
+    body: Optional[dict] = None,
+    file_upload_params: Optional[FileUploadParams] = None,
+) -> SlackResponse:
     """
     Sends an async request to slack API while handling rate limit errors.
 
     Args:
         client: The slack client.
         method: The method to use.
         http_verb: The HTTP method to use.
-        file_: A file path to send.
         body: The request body.
+        file_upload_params: An instance of FileUploadParams (for file upload APIs).
```

Comment:
```suggestion
        file_upload_params: An instance of FileUploadParams (for uploading using the file-upload API).
```
right?

Reply:
Improved doc string in commit 7c5b1ee65136fc655e92a3456301506de163b9e6